public/app/features/admin/ServerStats.tsx

```diff
@@ -81,14 +85,27 @@ export const ServerStats = () => {
             content={[
               { name: 'Organisations', value: stats?.orgs },
               { name: 'Users total', value: stats?.users },
-              { name: 'Active users in last 30 days', value: stats?.activeUsers },
               ...(config.featureToggles.displayAnonymousStats && stats?.activeDevices
+                ? [{ name: 'Active users in last 30 days', value: activeDevicesAndUsers }]
+                : [{ name: 'Active users in last 30 days', value: stats?.activeUsers }]),
+              ...(config.featureToggles.displayAnonymousStats && stats?.activeDevices && config.anonymousDeviceLimit
                 ? [
-                    { name: 'Active anonymous devices in last 30 days', value: stats?.activeDevices },
-                    { name: 'Active anonymous users in last 30 days', value: Math.floor(stats?.activeDevices / 3) },
+                    {
+                      name: 'Active anonymous users',
+                      value: `${Math.floor(stats?.activeDevices / 3)} / ${Math.floor(config.anonymousDeviceLimit / 3)}`,
```

Comment:
I would not display the anon User limit since it's going to be an unfamiliar number to the user and we'll enforce active devices only

Reply:
Okay, so only the Anonymous users? and not the limit?

Reply:
yes

Reply:
so only displaying the limit for the devices